dataset/dataset.go

```diff
@@ -621,8 +654,8 @@ func (c *Client) GetVersionMetadata(ctx context.Context, userAuthToken, serviceA
 	return
 }
 
-// GetDimensions will return a versions dimensions
-func (c *Client) GetDimensions(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, id, edition, version string) (m Dimensions, err error) {
+// GetVersionDimensions will return a list of dimension representations for a version
```

Comment:
Possibly: `GetVersionDimensions will return a list of dimensions for a given version of a dataset`